### PR TITLE
Readme.md updated for 22.09 release

### PR DESCRIPTION
**Pull request by orensbruli**
Signed-off-by: Esteban Martinena <orensbruli@gmail.com>

### CODE REVIEW

**Comment by marcoag** [comment]
This page should be specific to 22.09 but what I se in the instructions is still generic. 

- Missing binary installation
- If there are source installation instruction they should be specific on retrieving the latest 22.09 code, not the generic `https://raw.githubusercontent.com/open-rmf/rmf/main/rmf.repos`
- Docker containers are also using the `latest` tag, we should make this specifc.

**Comment by orensbruli** [comment]
We should also replicate the release https://github.com/open-rmf/rmf/releases/tag/21.09 for the 22.09, right?

File: README.md [hunk 1]
```diff
@@ -1,28 +1,17 @@
-# Robotics Middleware Framework (RMF)
-
-![](media/rmf_banner.png)
+# Open-RMF 22.09
 
 ![](https://github.com/open-rmf/rmf/workflows/build/badge.svg)
+
 ![Nightly](https://github.com/open-rmf/rmf/workflows/nightly/badge.svg)
```
Comment (marcoag) [inline README.md:5]: Are these specific for 22.09 or shall we remove them?

File: README.md [hunk 2]
```diff
@@ -140,15 +132,25 @@ colcon build --mixin release lld
...
 
+
+## Run RMF Demos
+
+Demonstrations of Open-RMF are shown in [rmf_demos](https://github.com/open-rmf/rmf_demos/).
+
+> **Note:** RMF Demos package cannot be installed on Humble distro because of an underlying [issue](https://github.com/open-rmf/rmf_demos/issues/166) with the release of a bad version of fastapi in jammy. You can install the package from [source](https://github.com/open-rmf/rmf/discussions/267).
+It is important to have `fastapi` installed via `pip` and not as an Ubuntu system package (ie, via `apt install`) for the reasons documented above.
+Please follow the instructions in the [Additional Dependencies](#additional-dependencies) section to install `fastapi` along with other dependencies needed to run Open-RMF demos.
+
```
Comment (marcoag) [inline README.md:143]: Maybe move this note or have a reference in the binaries installation section? My idea is to make people aware of this before they decide to go with binaries or source code installation.

Comment (orensbruli) [reply README.md]: @marcoag It is about a binary package so it has more sense to have this in the binary installation section. Done.

**Pull request closed without merging**